src/core/dbus-cgroup.c

```diff
@@ -347,6 +347,37 @@ static int property_get_ip_address_access(
         return sd_bus_message_close_container(reply);
 }
 
+static int property_get_bpf_foreign_program(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+        CGroupContext *c = userdata;
+        CGroupBPFForeignProgram *p;
+        int r;
+
+        r = sd_bus_message_open_container(reply, 'a', "s");
```

Comment:
please use a normalized type here i.e. use `a` with `(ss)`  here, and then pass the attach type string and the fs path as two separate strings

i.e.:
```c
r = sd_bus_message_open_container(reply, 'a', "(ss)");
```